src/components/OfflineIndicator.js

```diff
@@ -36,20 +36,22 @@ const defaultProps = {
     style: [],
 };
 
-const setStyles = (containerStyles, isSmallScreenWidth) => {
+const setStyles = (containerStyles, isSmallScreenWidth, styles) => {
```

Comment:
this function doesn't set anything, it returns needed styles, please change the name to 'getStyles'.